Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour {
+
+    [SerializeField]
+    private string sceneToLoad;
+    private bool isUnlocked = false;
+    [SerializeField]
+    SoundManager sound;
+
+    private bool isPlayerInTrigger;
+    
+
```

Comment:
Delete commented out code, remove all but one space between the variables and OnTriggerEnter2D, and put the serialize fields together at the top and the private bools below them.